### PR TITLE
Fix tutorial 3's wrong link and missing image

### PR DESCRIPTION
**Pull request by ThomasMiz**
# Summary of the PR
Fix tutorial 3's wrong link and missing image

### CODE REVIEW

File: website/docs/opengl/c1/3-hello-texture.md [hunk 1]
```diff
@@ -347,7 +347,7 @@ And now, some extra content to go with this tutorial!
 If you get curious about how the texture parameters work, let's learn it now!
 
 First, let's learn the structure of the command:
-```c#
+```c
```
Comment (curin) [inline website/docs/opengl/c1/3-hello-texture.md:350]:
```suggestion
```cs
```

**Review by curin — CHANGES_REQUESTED** [review]
Just a minor problem, otherwise looks good

File: website/docs/opengl/c1/3-hello-texture.md (hunk 1 above)
Comment (ThomasMiz) [reply website/docs/opengl/c1/3-hello-texture.md]: Lol, I should've used "replace all"

**Review by curin — APPROVED** [review]

File: website/docs/opengl/c1/3-hello-texture.md (hunk 1 above)
Comment (lumi2021) [reply website/docs/opengl/c1/3-hello-texture.md]: Lol imagine use replace all abd change everything to c

**Pull request merged**